vizro-core/docs/pages/user_guides/custom_actions.md

```diff
@@ -133,6 +133,10 @@ The following example shows how to create a custom action that shows the clicked
 
         Vizro().build(dashboard).run()
         ```
+
+        1. Just as for a normal Python function, the names of the arguments `show_species` and `points_data` are arbitrary and do not need to match on to the names of `inputs` in any particular way. 
```

Comment:
Do we need to emphasise that action function arguments are propagated in the following order:
1. static variables e.g. (`True` from the `my_custom_action(show_species=True)`),
2. dynamic variables e.g. `inputs`